src/sources/socket/udp.rs

```diff
@@ -53,6 +70,33 @@ pub fn udp(
             .expect("Failed to bind to udp listener socket");
         info!(message = "Listening.", address = %address);
```

Comment:
This should probably get moved after setting the buffer sizes, since vector doesn't actually listen until after those are set.